pages/linux/semanage-boolean.md

```diff
@@ -0,0 +1,21 @@
+# semanage boolean
+
+> Manage persistent SELinux boolean settings.
+> See also: `semanage`, `getsebool`, `setsebool`.
```

Comment:
```suggestion
> See also: `semanage` for managing SELinux policies, `getsebool` for checking the value of booleans, and `setsebool` for managing booleans temporarily.
```

Reply:
Yeah this works. Here's my small improvement:

```suggestion
> See also: `semanage` for managing SELinux policies, `getsebool` for checking boolean values, and `setsebool` for applying non-persistent boolean settings.
```

"non-persistent" is a bit more specific than "temporary" IMO.